From b14e969b5e577eabb7b246617a05d1dec642abe0 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Wed, 23 Dec 2020 09:27:22 +0800
Subject: [PATCH] Revert "Revert "arm64: dts: rockchip: Temporarily disable its
 for rk3568""

This reverts commit f818feceb5b682971b6f941a26cc0fb6ae5c7706.

TF-A do not support resume its right now.

Change-Id: Ibfd120242406feeae9171aa9d657b9cb6366e154
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 1eff2950c955..0ea22a0f4ca7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -369,6 +369,7 @@
 			msi-controller;
 			#msi-cells = <1>;
 			reg = <0x0 0xfd440000 0x0 0x20000>;
+			status = "disabled";
 		};
 	};
 
-- 
2.35.3

